tailwind.config.js

```diff
@@ -0,0 +1,17 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    "./src/**/*.{js,jsx,ts,tsx}",
+  ],
+  theme: {
+    extend: {
+      colors: {
+        'electric-blue': '#eefdff'
+      }, 
+      fontFamily: {
+        'inter': ['Inter']
```

Comment:
@aematei Change the `fontFamily` to `Inter, sans-serif` instead of just `inter`
<img width="681" alt="image" src="https://github.com/pogi7/luminosityleds-ui/assets/108915844/2070a677-3511-42ae-98bb-7c43e31d8daf">